cloudfoundry-client/src/main/java/org/cloudfoundry/client/v2/securitygroups/_RuleEntity.java

```diff
@@ -47,4 +47,25 @@
     @JsonProperty("protocol")
     abstract String getProtocol();
 
+    /**
+     * The control signal for icmp
+     */
+    @JsonProperty("type")
+    @Nullable
+    abstract Byte getType();
+
+    /**
+     * The control signal for icmp
+     */
+    @JsonProperty("code")
+    @Nullable
+    abstract Byte getCode();
```

Comment:
@s-bortolussi Should be an `Integer`.  I realize that this isn't accurate with regards to ICMP, but it is accurate with respects to the expected JSON payload.  I'll fix.